### DISCUSSION TITLE
Settings Guide

### DISCUSSION THREAD

**Discussion by origamifreak2** [beekeeper-studio/beekeeper-studio#2138] [Q&A] [answered]
I've added beekeeper studio's own app.db to my saved database connections, but I don't know what I can do with it, or even if I should do anything with it. I was hoping I could change some settings that might not be available in the UI. Is there any info on what you can do with beekeeper's own database?

**Comment by not-night-but** [comment, marked as answer]
@origamifreak2 Hey! The only settings that are currently available are available through the UI (mostly in the menu bar, or the status bar of certain views). We are working on adding much more configuration through a config.ini system, if you want to track the progress of that, look [here](https://github.com/beekeeper-studio/beekeeper-studio/pull/1929)

**Comment by origamifreak2** [comment]
Thanks :)
